frontend/src/routes/(frontend)/about/index.tsx

```diff
@@ -1,66 +1,63 @@
 import { component$ } from '@builder.io/qwik'
 import { DocumentHead, loader$ } from '@builder.io/qwik-city'
+import { getDatabase } from 'wildebeest/backend/src/database'
 import { getDomain } from 'wildebeest/backend/src/utils/getDomain'
+import { handleRequestGet as settingsHandleRequestGet } from 'wildebeest/functions/api/wb/settings/server/server'
+import { handleRequestGet as rulesHandleRequestGet } from 'wildebeest/functions/api/wb/settings/server/rules'
 import { Accordion } from '~/components/Accordion/Accordion'
-import { AccountCard } from '~/components/AccountCard/AccountCard'
+// import { AccountCard } from '~/components/AccountCard/AccountCard'
 import { HtmlContent } from '~/components/HtmlContent/HtmlContent'
-import { george } from '~/dummyData/accounts'
+import { ServerSettingsData } from '~/routes/(admin)/settings/server-settings/layout'
 import { Account } from '~/types'
 import { getDocumentHead } from '~/utils/getDocumentHead'
 import { instanceLoader } from '../layout'
 
 type AboutInfo = {
 	image: string
 	domain: string
-	contact: {
-		account: Account
-		email: string
-	}
-	rules: { id: string; text: string }[]
+	admin: Account & { email: string }
+	rules: { id: number; text: string }[]
 	extended_description: {
-		updated_at: string
 		content: string
 	}
 }
 
-export const aboutInfoLoader = loader$<Promise<AboutInfo>>(async ({ resolveValue, request, redirect }) => {
-	// TODO: properly implement loader and remove redirect
-	throw redirect(302, '/')
+export const aboutInfoLoader = loader$<Promise<AboutInfo>>(async ({ resolveValue, request, /*html,*/ platform }) => {
+	// TODO: properly implement loader and remove the following 404 throw
+	// throw html(404, getNotFoundHtml())
 
+	// TODO: fetching the instance for the thumbnail, but that should be part of the settings
 	const instance = await resolveValue(instanceLoader)
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- shouldn't be necessary after rebase
+	const database = getDatabase(platform as any)
+
+	const brandingDataResp = await settingsHandleRequestGet(database)
+	let brandingData: ServerSettingsData | null
+	try {
+		brandingData = await brandingDataResp.json()
+	} catch {
+		brandingData = null
+	}
+
+	const rulesResp = await rulesHandleRequestGet(database)
+	let rules: { id: number; text: string }[] = []
+	try {
+		rules = await rulesResp.json()
+	} catch {
+		rules = []
+	}
+
+	// TODO: get admin account here when we've implemented that
+	const admin = null as unknown as Account & { email: string }
```

Comment:
@xtuc as discussed we'll get the contact and email on setup, for now I am just setting this to null and not showing anything regarding the admin in the about UI